cluster-autoscaler/cloudprovider/clusterapi/clusterapi_controller.go

```diff
@@ -551,11 +553,10 @@ func (c *machineController) nodeGroupForNode(node *corev1.Node) (*nodegroup, err
 			if err != nil {
 				return nil, fmt.Errorf("failed to build nodegroup for node %q: %v", node.Name, err)
 			}
-			// We don't scale from 0 so nodes must belong
-			// to a nodegroup that has a scale size of at
-			// least 1.
-			if nodegroup.MaxSize()-nodegroup.MinSize() < 1 {
-				return nil, nil
+			if !nodegroup.scalableResource.CanScaleFromZero() {
```

Comment:
same that https://github.com/openshift/kubernetes-autoscaler/pull/137/files#r396385396